java/src/client/roll/RollController.java

```diff
@@ -19,6 +21,7 @@
 	public RollController(IRollView view, IRollResultView resultView) {
 		super(view);
 		setResultView(resultView);
+		roller = new Dice(2,12);
```

Comment:
if you do it just like this, then the percentages of numbers rolled will be more equal than 2 die... like 2 has the same probability of being rolled as a 7. I would recommend making two die from 1-6 and adding up their rolls. What do you think?